mod/homepage/template/self/homepage_numbers.haml

```diff
@@ -0,0 +1,21 @@
+.row.m-0.p-0.transparent-background-white.nodblclick.slide-container.border-top
+  .col-12.text-center.p-3
+    %h1.font-weight-light.m-4 WikiRate in numbers
+  .mx-auto.mt-4
+    .row
+      .col-12.col-md-6
+        - categories.each_with_index do |cat,i|
+          .d-flex.mb-5.mr-4.justify-content-end
+            .text-right.mx-3
+              %h1.font-weight-normal_count-ele{id: "count-ele-#{i}"}
+                = cat.count
+              %h5.font-weight-normal.text-muted
+                = cat.title
+            .drop-shadow.rounded-circle.bg-white.image-box.small.m-2{class: "#{cat.color}-color"}
+              = icon_tag(cat.icon, class: "md-24 p-3")
+      .d-none.d-md-inline.col-5.mt-4
+        .carousel.carousel-fade.slide.drop-shadow{"data-ride" => "carousel"}
```

Comment:
missing `.` near `_count-ele`
Should be `%h1.font-weight-normal._count-ele{id: "count-ele-#{i}"}`